internal/compiler/passes/inlining.rs

```diff
@@ -486,6 +486,12 @@ fn component_requires_inlining(component: &Rc<Component>) -> bool {
         return true;
     }
 
+    // the focus_item pass needs to refer to elements that are focusable, if it is not inline
+    // it is not possible to refer to them in an  Expression::ElementReference
+    if matches!(&root_element.borrow().base_type, ElementType::Builtin(b) if b.accepts_focus) {
```

Comment:
That's a bummer. Why is it not possible?

Reply:
```
MyTextInput := TextInput { }

//...
Foo := window { 
   //..
      e1 := MyTextInput {}
      e2 := MyTextInput {}
}
```

Because the element is the FocusScope or InputText  which is within a subcomponent,  and our  Expression::ElementReference can't point to it.  It can point to e1 or e2,  but that element is not a builtin element and latter pass can't work with that anymore.

Reply:
Ah right, there's on `ElementRc` for that.